srcpkgs/lainsafecli/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'lainsafecli'
+pkgname=lainsafecli
+version=0.4
+revision=1
+archs="noarch"
+create_wrksrc=yes
```

Comment:
Not needed, actually bad

Reply:
You mean `create_wrksrc`? If I remove it i guest this:
`=> ERROR: lainsafecli-0.4_1: cannot access wrksrc directory [/builddir/lainsafecli-0.4]`

Reply:
Oh, yeah. So, you should make it be `wrksrc="lainsafecli-${version}"` then

Reply:
It still get it. Maybe because the tar file?